api/src/test/java/org/openmrs/module/bedmanagement/BedManagementServiceTest.java

```diff
@@ -56,9 +64,10 @@ public void getAllAdmissionLocations_gets_locations_that_support_admission() {
 
         BedManagementServiceImpl bedManagementService = new BedManagementServiceImpl();
         bedManagementService.setDao(bedManagementDao);
+        bedManagementService.setLocationService(locationService);
 
-        List<AdmissionLocation> wards = bedManagementService.getAllAdmissionLocations();
-        Assert.assertSame(expectedWards, wards);
+        List<AdmissionLocation> wards = bedManagementService.getAdmissionLocations();
```

Comment:
This test
* Is not a context-sensitive test.
* Doesn't really test anything useful.
* Is the only place in the module code base that uses the now reimplemented `getAdmissionLocationsBy(..)`.
  * Could you confirm this, and **if I'm correct**, remove the now redundant methods `BedManagementDaoImpl.getAdmissionLocationsBy(..)`, `BedManagementDaoImpl.getPhysicalLocationsByLocationTag(..)`?
  * We could still keep `BedManagementDaoImpl.getPhysicalLocationsByLocationTagAndParentLocation(..)` since it seems that this is not covered anywhere else. It might still be useful one day.

So either remove this test altogether, or make it context-sensitive and test something useful using your new/refactored API.